components/address/Address.tsx

```
@@ -135,19 +134,17 @@ export default function Address(props: Readonly<Props>) {
             &nbsp;&nbsp;
-            <a
+            <Link
               className="decoration-none decoration-hover-underline"
-              href={getProfileLink()}
-            >
+              href={getProfileLink()}>
               <span
                 className={`${styles.consolidationDisplay} ${
                   props.isUserPage ? styles.consolidationDisplayUserPage : ""
                 }`}
                 dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)